server/webapp/WEB-INF/rails.new/spec/views/admin/templates/index_html_spec.rb

```diff
@@ -283,16 +351,38 @@
     end
   end
 
-  it "should disable the delete button next to the template name for template admin" do
+  it "should display the delete button next to the template name for template admin" do
```

Comment:
Hmm. I don't remember discussing whether template admins can now, suddenly, delete a template. I suppose it makes sense, given that they can create it? The only thing they cannot do is edit its permissions? It seems like existing template admins are now a little more powerful, with no way to reduce that.